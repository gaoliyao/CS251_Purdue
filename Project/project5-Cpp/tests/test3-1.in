3
50 300
AAA ABO 1.00
AAB AAW 2.00
AAC ABN 3.00
AAD AAR 4.00
AAE ABF 5.00
AAF AAI 6.00
AAG ABT 7.00
AAH ABI 8.00
AAI ABO 9.00
AAJ AAZ 10.00
AAK ABC 11.00
AAL AAO 12.00
AAM AAK 13.00
AAN AAX 14.00
AAO ABD 15.00
AAP AAW 16.00
AAQ AAF 17.00
AAR AAZ 18.00
AAS ABP 19.00
AAT ABO 20.00
AAU AAI 21.00
AAV ABV 22.00
AAW ABD 23.00
AAX ABB 24.00
AAY AAZ 25.00
AAZ AAN 26.00
ABA ABO 27.00
ABB ABP 28.00
ABC AAI 29.00
ABD ABP 30.00
ABE ABH 31.00
ABF AAA 32.00
ABG AAN 33.00
ABH AAY 34.00
ABI AAS 35.00
ABJ ABS 36.00
ABK ABH 37.00
ABL AAP 38.00
ABM ABF 39.00
ABN AAZ 40.00
ABO AAJ 41.00
ABP ABQ 42.00
ABQ AAD 43.00
ABR ABI 44.00
ABS AAV 45.00
ABT ABB 46.00
ABU ABP 47.00
ABV ABU 48.00
ABW AAS 49.00
ABX ABF 50.00
AAE AAS 51.00
AAL ABH 52.00
ABR ABA 53.00
AAY ABL 54.00
ABK AAR 55.00
AAU ABK 56.00
ABF ABT 57.00
AAG ABB 58.00
ABE AAV 59.00
AAM AAR 60.00
ABX AAR 61.00
AAD AAU 62.00
AAT AAP 63.00
ABA AAZ 64.00
ABH ABN 65.00
AAJ ABC 66.00
AAR ABJ 67.00
AAC ABM 68.00
ABN ABJ 69.00
ABJ ABU 70.00
AAN AAR 71.00
AAT AAX 72.00
ABP AAY 73.00
AAA ABU 74.00
ABP AAP 75.00
ABP ABV 76.00
AAI AAC 77.00
ABB AAZ 78.00
ABL ABV 79.00
ABC ABN 80.00
ABJ AAT 81.00
AAZ AAW 82.00
AAR ABM 83.00
ABP ABL 84.00
AAL AAP 85.00
AAU AAE 86.00
ABP AAX 87.00
AAB ABM 88.00
AAW ABS 89.00
AAF ABI 90.00
AAL ABW 91.00
ABH AAV 92.00
AAA AAM 93.00
ABU ABL 94.00
AAJ ABA 95.00
ABD ABU 96.00
ABV AAE 97.00
AAU AAP 98.00
ABS AAL 99.00
AAE AAH 100.00
ABD ABA 101.00
AAO AAU 102.00
AAK AAX 103.00
AAL AAQ 104.00
AAJ AAY 105.00
AAO ABQ 106.00
ABT AAR 107.00
AAH ABR 108.00
AAR AAS 109.00
ABN ABL 110.00
ABJ ABH 111.00
AAB ABN 112.00
ABP ABE 113.00
AAS AAH 114.00
AAC AAT 115.00
AAY AAN 116.00
ABR ABJ 117.00
ABF AAC 118.00
AAL ABV 119.00
ABV AAG 120.00
AAO AAE 121.00
AAB AAV 122.00
AAY AAW 123.00
AAE ABP 124.00
ABQ ABS 125.00
ABD ABE 126.00
ABD ABG 127.00
AAV AAW 128.00
AAO ABN 129.00
ABD AAQ 130.00
AAJ AAF 131.00
AAG AAD 132.00
AAM AAD 133.00
AAJ AAM 134.00
ABV ABI 135.00
ABI AAE 136.00
ABB ABD 137.00
AAA AAI 138.00
AAJ ABF 139.00
ABO ABE 140.00
AAU ABJ 141.00
AAX ABF 142.00
ABR AAE 143.00
AAV ABR 144.00
ABV AAA 145.00
AAL AAI 146.00
AAN AAL 147.00
ABS ABX 148.00
AAR AAW 149.00
ABC AAR 150.00
ABE ABL 151.00
AAA AAX 152.00
AAU AAG 153.00
ABB ABQ 154.00
ABQ AAC 155.00
AAX ABL 156.00
AAJ AAK 157.00
AAW ABE 158.00
AAD AAL 159.00
AAN AAC 160.00
AAM AAY 161.00
AAL ABB 162.00
ABL AAF 163.00
ABA AAE 164.00
ABD AAF 165.00
AAX AAM 166.00
ABS AAY 167.00
ABJ AAQ 168.00
ABE AAO 169.00
AAI AAY 170.00
AAR ABH 171.00
AAL ABA 172.00
AAG ABX 173.00
AAB AAL 174.00
ABV AAO 175.00
ABM ABJ 176.00
AAW AAN 177.00
AAN AAQ 178.00
ABN AAA 179.00
ABH AAT 180.00
AAP ABR 181.00
ABS ABG 182.00
ABB AAF 183.00
AAI AAW 184.00
ABP AAQ 185.00
AAV ABN 186.00
AAA ABI 187.00
AAT AAJ 188.00
AAG AAL 189.00
AAN ABC 190.00
ABS AAP 191.00
ABK ABQ 192.00
AAX ABE 193.00
ABA AAC 194.00
ABM ABK 195.00
AAY ABF 196.00
AAC ABU 197.00
AAU ABN 198.00
AAV AAX 199.00
AAZ AAI 200.00
AAI ABH 201.00
ABW ABK 202.00
AAD ABO 203.00
ABA ABN 204.00
ABI AAB 205.00
AAW AAK 206.00
ABU ABB 207.00
ABR AAA 208.00
AAZ AAQ 209.00
ABN AAR 210.00
AAH AAF 211.00
AAO ABB 212.00
AAM AAZ 213.00
AAE ABW 214.00
ABW ABC 215.00
AAJ AAB 216.00
ABO AAH 217.00
ABF ABI 218.00
AAI AAG 219.00
AAA ABX 220.00
AAD ABP 221.00
AAS AAM 222.00
ABU ABG 223.00
ABN AAX 224.00
AAI AAB 225.00
ABW AAU 226.00
ABA AAT 227.00
ABL ABO 228.00
AAR ABQ 229.00
AAU AAC 230.00
AAT AAN 231.00
AAJ AAA 232.00
ABV AAT 233.00
AAI ABX 234.00
ABO ABN 235.00
AAZ ABM 236.00
AAY AAQ 237.00
AAL ABG 238.00
AAT AAL 239.00
AAF ABU 240.00
ABE ABQ 241.00
ABM ABX 242.00
ABK ABD 243.00
AAR AAJ 244.00
AAW ABR 245.00
ABO AAV 246.00
AAM ABW 247.00
AAU ABH 248.00
AAG AAN 249.00
ABF AAZ 250.00
AAY ABK 251.00
AAX AAG 252.00
ABF AAO 253.00
AAI AAR 254.00
ABR ABB 255.00
ABA ABB 256.00
AAP AAA 257.00
AAU AAF 258.00
AAV ABI 259.00
AAF ABR 260.00
ABQ AAP 261.00
AAU AAX 262.00
ABQ ABU 263.00
ABG AAP 264.00
ABP ABA 265.00
AAO AAN 266.00
AAG ABF 267.00
ABL AAO 268.00
AAY ABU 269.00
AAV AAS 270.00
AAN ABQ 271.00
ABP AAF 272.00
ABO AAF 273.00
AAX ABV 274.00
AAA AAF 275.00
AAM AAN 276.00
ABH AAE 277.00
AAE ABL 278.00
AAN ABT 279.00
AAK ABB 280.00
AAI ABK 281.00
ABH ABI 282.00
ABG AAE 283.00
AAC ABT 284.00
ABW ABT 285.00
AAC ABP 286.00
AAB ABC 287.00
ABH AAB 288.00
AAO AAS 289.00
AAG AAT 290.00
AAQ AAW 291.00
ABU ABC 292.00
ABX ABT 293.00
ABV AAJ 294.00
ABF ABG 295.00
ABL ABT 296.00
AAM ABL 297.00
ABC ABR 298.00
ABR AAR 299.00
ABV ABC 300.00
AAU
