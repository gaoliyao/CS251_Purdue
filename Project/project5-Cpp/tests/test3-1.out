AAU
AAI
AAF
AAQ
ABC
AAK
AAM
ABO
AAA
ABF
AAE
ABM
ABX
AAJ
AAZ
AAN
AAX
ABB
ABP
AAS
ABI
AAH
ABR
ABW
ABD
AAO
AAL
AAW
AAB
AAP
ABL
ABQ
ABU
ABV
AAV
ABS
ABJ
ABT
AAG
ABG
AAR
AAD
AAY
ABH
ABE
ABK
ABN
AAC
AAT
ABA
